data/resources/ui/content-message-sticker.blp

```diff
@@ -4,12 +4,19 @@ template MessageSticker : .MessageBase {
   layout-manager: BinLayout {};
 
   Overlay {
```

Comment:
I think this should actually be a box now?

Reply:
I think it have to be an Overlay or at least it shouldn't be a GtkBox, cause it look like that with GtkBox (the message is outgoing)
![screenshot](https://user-images.githubusercontent.com/110191114/221619964-d805a83c-3c35-44f6-af2e-2e415077317f.png)

Reply:
and it should looks like that 
![screenshot](https://user-images.githubusercontent.com/110191114/221620469-14c50302-ced9-4c33-ac21-9b254606e3db.png)

Reply:
> and it should looks like that

I think that we want the reply widget to be attached to the sticker on the right, just like Telegram Desktop.

Reply:
> I think that we want the reply widget to be attached to the sticker on the right, just like Telegram Desktop.

Personally I prefer the way it looks on the mobile, but fine I'll follow to your suggestions.

Reply:
> Personally I prefer the way it looks on the mobile

I don't like it because it's too far away from the actual sticker. Maybe we can use that style on mobile in the future, but for now let's focus on desktop.